doc/nrf/known_issues.rst

```diff
@@ -1432,6 +1432,16 @@ KRKNWK-8842: MPSL does not support nRF21540 revision 1 or older
 
   **Workaround:** Check nordicsemi.com for the latest information on availability of the product version of nRF21540.
 
+.. rst-class:: v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
+
+DRGN-16642: If the HFCLK is requested from the application and radio notifications on ACTIVE are used before the HFCLK is released again, MPSL may assert.
+**Workaround**: use radio notifications on nACTIVE.
```

Comment:
Please indent and format the line containing "**Workaround**" as done for other known issues. That ensures proper formatting when the documentation is generated.